src/Controller/DeleteUserController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Entity\User;
+use App\Repository\UserRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route (path="/api/users")
+ */
+class DeleteUserController extends AbstractController
+{
+    private UserRepository $userRepository;
+
+    private EntityManagerInterface $entityManager;
+
+    public function __construct(UserRepository $userRepository, EntityManagerInterface $entityManager)
+    {
+        $this->userRepository = $userRepository;
+        $this->entityManager = $entityManager;
+    }
+
+
+    /**
+     * @Route (path="/{id}", methods={"DELETE"})
+     */
+    public function deleteUserAction(int $id): Response
+    {
+        $deletedUser = $this->userRepository->findOneBy(['id' => $id]);
```

Comment:
call this `userToDelete` or `user` because this name suggests it's already deleted